### PR TITLE
Implement UserDropdownMenu component

### PR DESCRIPTION
**Pull request by reynard2007**
Task #8

### CODE REVIEW

**Comment by vikasrohit** [comment]
Thanks @reynard2007 Merging branch. Final fixes would be done separately.

File: components/UserDropdownMenu/UserDropdownMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+require('./UserDropdownMenu.scss')
```
Comment (nlitwin) [inline components/UserDropdownMenu/UserDropdownMenu.jsx]: Same comments as https://github.com/appirio-tech/react-components/commit/3118fed8f68e8ca4a511142d2cbdf982f9344a6d

File: components/UserDropdownMenu/UserDropdownMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  ],
+  [
+    { label: 'Help', link: 'javascript:;', id: 0 }
+  ],
+  [
+    { label: 'Log out', link: 'javascript:;', id: 0 }
+  ]
+]
+
+const UserDropdownMenu = {
```
Comment (nlitwin) [inline components/UserDropdownMenu/UserDropdownMenu.jsx]: ES6 class for stateful components, see https://github.com/appirio-tech/react-components/commit/3118fed8f68e8ca4a511142d2cbdf982f9344a6d

File: components/UserDropdownMenu/UserDropdownMenu.scss [hunk 3]
```diff
@@ -0,0 +1,64 @@
+@import "work/work-includes";
```
Comment (nlitwin) [inline components/UserDropdownMenu/UserDropdownMenu.scss]: Use topcoder includes instead

File: components/UserDropdownMenu/UserDropdownMenu.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  getInitialState () {
+    return { isLoggedIn: true }
+  },
+  render() {
+    let dom
+
+    if (!this.state.isLoggedIn) {
+      dom = <div><button>Log in</button><button>Join</button></div>
```
Comment (nlitwin) [inline components/UserDropdownMenu/UserDropdownMenu.jsx]: Can we save these two different values for `dom` as descriptive variables, and then `return this.state.isLoggedIn ? dropdown : accountButtons` (doesn't have to be those exact `var` names

File: components/UserDropdownMenu/UserDropdownMenuExamples.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+'use strict'
+
```
Comment (nlitwin) [inline components/UserDropdownMenu/UserDropdownMenuExamples.jsx]: Same comment for use strict/imports

**Pull request merged**